### PR TITLE
added triangle

### PR DESCRIPTION
**Pull request by kinoko-neko**
Added basic triangle info.

### CODE REVIEW

**Review by hrmnpfg — CHANGES_REQUESTED** [review]
you forgor

File: src/com/oneofever/commands/Triangle.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    com.oneofever.shapes.Triangle triangle = new com.oneofever.shapes.Triangle(argType, value);
+
+    System.out.println("side = " + triangle.getSide());
+    System.out.println("height = " + triangle.getHeight());
+    System.out.println("area = " + triangle.getArea());
+  }
+
+  public String usage() {
+    return "Usage:\n" + //
+            "\tsquare {side | height | area} <value>";
```
Comment (Acors24) [inline src/com/oneofever/commands/Triangle.java:53]: "square" 🤦🏻

File: src/com/oneofever/commands/Triangle.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    com.oneofever.shapes.Triangle triangle = new com.oneofever.shapes.Triangle(argType, value);
+
+    System.out.println("side = " + triangle.getSide());
+    System.out.println("height = " + triangle.getHeight());
+    System.out.println("area = " + triangle.getArea());
+  }
+
+  public String usage() {
+    return "Usage:\n" + //
+            "\ttriange{side | height | area} <value>";
```
Comment (Acors24) [inline src/com/oneofever/commands/Triangle.java:53]: "triange" 🤦🏻

File: src/com/oneofever/commands/Triangle.java (hunk 1 above)
Comment (kinoko-neko) [reply src/com/oneofever/commands/Triangle.java]: i rember

**Pull request merged**